### PR TITLE
Remove unused matrixVP to prevent shader error

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview
After fixing an issue in #6476, it appears to have caused a second shader error.
This math / variable doesn't even appear to be used, so I've removed it from both shaders.

## Changes
- Fixes: #6577 

## Verification
Ensure the features that use these shaders (Lost Tracking Service) still render correctly.

### CODE REVIEW

**Comment by davidkline-ms** [comment]
looks good to me... i would appreciate a shader expert to give a review

File: Assets/MixedRealityToolkit.Extensions/LostTrackingService/Assets/SR_Triangles_Object_Space.shader [hunk 1]
```diff
@@ -132,7 +132,6 @@ SubShader {
 
     FragmentInput vxOut[Geo_Max_Out_Vertices];
```
Comment (Troy-Ferrell) [inline Assets/MixedRealityToolkit.Extensions/LostTrackingService/Assets/SR_Triangles_Object_Space.shader]: can you fix Geo_Max_Out_Vertices as well? ifndef seems unnecessary?

File: Assets/MixedRealityToolkit.Extensions/LostTrackingService/Assets/SR_Triangles.shader [hunk 2]
```diff
@@ -135,11 +135,9 @@ SubShader {
 
     FragmentInput vxOut[Geo_Max_Out_Vertices];
     int stripVxCount[Geo_Max_Out_Vertices];
```
Comment (Troy-Ferrell) [inline Assets/MixedRealityToolkit.Extensions/LostTrackingService/Assets/SR_Triangles.shader]: Also while here can you clean up the file?
Not readable spacing/indenting especially at top of file, commented lines such as //bool _Color_Map_Enable_; tags are duplicated. Also some of the defines like No_Depth_Write?

Was this created via shader graph of something?

**Comment by Troy-Ferrell** [comment]
> looks good to me... i would appreciate a shader expert to give a review

Skimming through this, main thing that scares me is the use of a geometry shader and that double while loop inside of it. At least maxvertexcount is only 3.

This looks like it was generated shader code from some tool like shadergraph. It's a hard to follow what the shader is doing, or at least supposed to do, and if it could be done in a more performant way

File: Assets/MixedRealityToolkit.Extensions/LostTrackingService/Assets/SR_Triangles.shader (hunk 2 above)
Comment (keveleigh) [reply Assets/MixedRealityToolkit.Extensions/LostTrackingService/Assets/SR_Triangles.shader]: Can do!

Not sure the source. @railboy?
